packages/compiler/src/core/types.ts

```diff
@@ -609,7 +609,7 @@ export interface StringTemplateSpanValue extends BaseType {
 
 export interface Tuple extends BaseType {
   kind: "Tuple";
-  node: TupleExpressionNode | ArrayLiteralNode;
+  node?: TupleExpressionNode | ArrayLiteralNode;
```

Comment:
Builds off of #7069

Reply:
ah oops missed one

Reply:
No worries! Gives me a chance to fight CI for a bit and learn more about the codebase 😄